config/initializers/thredded.rb

```diff
@@ -1,36 +1,22 @@
 # frozen_string_literal: true
-# Thredded configuration
 
-# ==> User Configuration
-# The name of the class your app uses for your users.
-# By default the engine will use 'User' but if you have another name
-# for your user class - change it here.
+# Set the default user class to make sure Thredded adds its methods to it.
 Thredded.user_class = 'User'
 
-# User name column, used in @mention syntax and should be unique.
-# This is the column used to search for users' names if/when someone is @ mentioned.
+# Use User#login for the @mention syntax and to present the name of the user. Note that this column
+# must be unique.
 Thredded.user_name_column = :login
 
-# User display name method, by default thredded uses the user_name_column defined above
-# You may want to use :to_s or some more elaborate method
-#Thredded.user_display_name_method = :to_s
+# Configure the method name to fetch the current user.
+Thredded.current_user_method = :current_user
 
-# The path (or URL) you will use to link to your users' profiles.
-# When linking to a user, Thredded will use this lambda to spit out
-# the path or url to your user. This lambda is evaluated in the view context.
-Thredded.user_path = lambda do |user|
-  "/#{user.to_param}"
-end
+# Lambda to generate a URL to the user page.
+Thredded.user_path = ->(user) { '/' + user.to_param }
 
-# This method is used by Thredded controllers and views to fetch the currently signed-in user
-Thredded.current_user_method = :"current_#{Thredded.user_class.name.underscore}"
-
-# User avatar URL. rb-gravatar gem is used by default:
-Thredded.avatar_url = ->(user) { user&.pic&.pic&.url(:album) || ActionController::Base.helpers.image_url('/default/o-pic_white.svg') }
-
-# ==> Permissions Configuration
-# By default, thredded uses a simple permission model, where all the users can post to all message boards,
-# and admins and moderators are determined by a flag on the users table.
+# Configure which URL to use for an avatar.
+Thredded.avatar_url = ->(user) {
+  user.avatar_url(variant: :album) || UsersHelper.no_avatar_path
```

Comment:
`|| UsersHelper.no_avatar_path` is redundant here.

Reply:
No, `user.avatar_url` will return `nil` when there is no avatar. See the spec below.

Reply:
Ah gotcha, I misread as `user_avatar_url` and was looking at the helper which has the same logic.